app/src/main/res/drawable/exit_ic.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
ic_exit

Reply:
done in this pr: https://github.com/iTarley/QuizApp-SpaceIntern/pull/6